### PR TITLE
added a new js file

### PR DESCRIPTION
**Pull request by joshua-alex**
I added a js file, that's it man.

### CODE REVIEW

**Review by blumdani — CHANGES_REQUESTED** [review]
Rubbish, do it over.

**Comment by raulmartinez94** [comment]
awesome!

File: jwilson.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Something, it could be anything.');
```
Comment (UrazielNomolos) [inline jwilson.js:1]: This is a comment. I am commenting on this line. Feel free to feel commented on.

**Review by blumdani — APPROVED** [review]
10/10. Best code. Would review again

**Comment by clairely-u** [comment]
neato

**Pull request merged**